tests/stdlib/topenssl.nim

```diff
@@ -0,0 +1,16 @@
+import openssl
+
+# bug #16308
+block:
+  var rsa: PRSA
+  var file: File
+
+  if not file.open("topenssl_rsa_public_key.pem"):
+    raise newException(Exception, "Error opening file")
+
+  rsa = PEM_read_RSAPUBKEY(file, nil, nil, nil)
+  if rsa == nil:
+    raise newException(Exception, "Error reading public key")
+  RSA_free(rsa)
+  file.close()
+  echo "ok"

```

Comment:
```suggestion
```
`echo` is unnecessary and will cause tests failed. `testament` will run them when not specifying spec.
see https://nim-lang.github.io/Nim/testament.html